13-TeV-examples/uproot_python/HyyAnalysisNew.ipynb

```diff
@@ -0,0 +1,771 @@
+    "\n",
+    "Notebooks are web applications that allow you to create and share documents that can contain for example:\n",
+    "1. live code\n",
+    "2. visualisations\n",
+    "3. narrative text\n",
+    "\n",
+    "This analysis loosely follows the discovery of the Higgs boson by ATLAS https://arxiv.org/pdf/1207.7214.pdf (mostly Section 5 and 5.1)\n",
+    "\n",
+    "By the end of this notebook you will be able to:\n",
+    "1. rediscover the Higgs boson yourself!\n",
+    "2. know some general principles of a particle physics analysis\n",
+    "\n",
+    "Feynman diagram pictures are borrowed from our friends at https://www.particlezoo.net"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<CENTER><img src=\"images/feynman_diagrams/Hyy_feynman.png\" style=\"width:40%\"></CENTER>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='contents'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Contents: \n",
+    "\n",
+    "[Running a Jupyter notebook](#running) <br />\n",
+    "[First time setup on your computer (no need on mybinder)](#setup_computer) <br />\n",
+    "[To setup everytime](#setup_everytime) <br />\n",
+    "[Lumi, fraction, file path](#fraction) <br />\n",
+    "[Samples](#samples) <br />\n",
+    "[Changing a cut](#changing_cut) <br />\n",
+    "[Applying a cut](#applying_cut) <br />\n",
+    "[Plotting](#plotting) <br />\n",
+    "[What can you do to explore this analysis?](#going_further) <br />"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='running'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running a Jupyter notebook\n",
+    "\n",
+    "To run the whole Jupyter notebook, in the top menu click Cell -> Run All.\n",
+    "\n",
+    "To propagate a change you've made to a piece of code, click Cell -> Run All Below.\n",
+    "\n",
+    "You can also run a single code cell, by clicking Cell -> Run Cells, or using the keyboard shortcut Shift+Enter."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='setup_computer'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## First time setup on your computer (no need on mybinder)\n",
+    "This first cell only needs to be run the first time you open this notebook on your computer. \n",
+    "\n",
+    "If you close Jupyter and re-open on the same computer, you won't need to run this first cell again.\n",
+    "\n",
+    "If you open on binder, you don't need to run this cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
```

Comment:
Can you put the specific versions here? To avoid future problems running this one.